src/make-comment.ts

```diff
@@ -39,18 +39,18 @@ const makePullRequestComment = async (message: string | undefined, commentData:
 
     const body = message ?? (await constructComment(commentData));
 
-    if (!botComment) {
-      octokit.issues.createComment({
+    if (botComment) {
+      octokit.issues.updateComment({
         owner: owner,
         repo: repo,
-        issue_number: pullRequestNumber,
+        comment_id: botComment.id,
```

Comment:
also why

Reply:
same as above! the order of the if/else is swapped not the fields inside each block